File: src/main/java/rk/cinema/repository/MovieRepository.java
package rk.cinema.repository;

import io.micronaut.data.jdbc.annotation.JdbcRepository;
import io.micronaut.data.model.query.builder.sql.Dialect;
import io.micronaut.data.repository.CrudRepository;
import rk.cinema.model.Movie;

@JdbcRepository(dialect = Dialect.POSTGRES)
public interface MovieRepository extends CrudRepository<Movie, Long> {
}

File: src/main/kotlin/rk/powermilk/cinema/repository/MovieRepository.kt
package rk.powermilk.cinema.repository

import io.micronaut.data.jdbc.annotation.JdbcRepository
import io.micronaut.data.model.query.builder.sql.Dialect
import io.micronaut.data.repository.CrudRepository
import rk.powermilk.cinema.model.Movie

@JdbcRepository(dialect = Dialect.POSTGRES)
interface MovieRepository : CrudRepository<Movie, Long>
